examples/darwin-framework-tool/commands/tests/TestCommandBridge.h

```diff
@@ -119,8 +158,9 @@ class TestCommandBridge : public CHIPCommandBridge,
 
         // Invalidate our existing CASE session; otherwise getConnectedDevice
         // will just hand it right back to us without establishing a new CASE
-        // session.
+        // session when a reboot is done on the all-clusters-app
```

Comment:
```suggestion
        // session when a reboot is done on the server.
```
Could be any of the server apps...

Reply:
Updated